R/formatters_var_labels.R

```diff
@@ -43,7 +43,9 @@ col_labels <- function(x, fill = FALSE) {
     return(character(0L))
   }
 
-  labels <- lapply(x, attr, "label")
+  labels <- lapply(x, function(col) {
+    unname(attr(col, "label"))
+  })
```

Comment:
Would this do the same trick?
```suggestion
  labels <- lapply(lapply(x, attr, "label"), unname)
```

Notwithstanding, I would rather remove the source of the problem than deal with such consequences. The label should not be named in the first place.

Reply:
I think labels can be named or unnamed. Why should they be not named?

Reply:
Why would they be named? What would be the use?

Reply:
It becomes easier to pass the labels without the data. Because the names contain the columns they are bound to.

Reply:
Ok, I see now that `value` is permitted to be named. I believe those names should be dropped in the setter.